src/lib/web-publisher-info.js

```diff
@@ -0,0 +1,38 @@
+import { database_pool } from './database.js';
+
+class  PublisherInfo {
+  static async getInfo(publisherId) {
+    const client = await database_pool.connect();
+    const publisherSqlQuery = `
+     SELECT
+        name,
+        url
+      FROM publisher
+      WHERE id=$1 LIMIT 1
+    `;
+
+    const resPublisher = await client.query(publisherSqlQuery, [publisherId]);
+
+
+    const publisherFeedsSqlQuery = `
+     SELECT
+        name,
+        url
+      FROM publisher_feed
+      WHERE publisher_id=$1
+    `;
+
+    const resFeeds = await client.query(publisherFeedsSqlQuery, [publisherId]);
+
+    client.release();
```

Comment:
Should this be in a try ... catch .... finally so the client is always released? like other code

Reply:
Also, I'm not sure about the details of this data base driver; I'm not sure how wise it is to refer to results after the client is released. I've seen problems with this and multi-threading on other database drivers. I'd put all database work in a big try .... finally block, which should make this fool proof for crashes that cause problems elsewhere. (We might have already seen one issues around that)